llarp/quic/endpoint.hpp

```diff
@@ -111,7 +111,7 @@ namespace llarp::quic
 
     // Called to handle an incoming packet
     void
-    handle_packet(const Packet& p);
+    handle_packet(Packet p);
```

Comment:
This would be better as an rvalue reference `Packet&& p` so that we are still passing it by reference rather than needing to copy it (there is internal ngtcp2 C stuff in here that likely isn't moveable, so taking it by value will involve at least *some* copying).  Whereas if you take it as a `Packet&&` you *have* to move it when calling, but we still take it as a reference so don't have to copy anything.

Reply:
hah, i knew it